integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -234,11 +277,20 @@ public void unlockBlockTest2() throws IOException {
     for (int k = 0; k < numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
       Assert.assertTrue(tFile.isInMemory());
-      TachyonByteBuffer tBuf = tFile.readByteBuffer(0);
-      Assert.assertNotNull(tBuf);
-      tBuf = tFile.readByteBuffer(0);
-      Assert.assertNotNull(tBuf);
-      tBuf.close();
+      is = tFile.getInStream(ReadType.CACHE);
+      if (is.markSupported()) {
```

Comment:
The logic of original code is to read block 0 twice. By using InStream, we need to reset/rewind the stream to the beginning and read again. So markSupported is required to test whether mark and reset methods works. Of course, we could replace this with creating a new InStream and read.